Releasing.md

```diff
@@ -170,12 +170,13 @@ For the body of the post, just copy this checklist and again replace all occurre
 <p>o Ensure that the bundle files are updated to include any changes to the release branch by running <code>npm run bundle</code> in gutenberg-mobile release branch and committing any changes. </p>
 <!-- /wp:paragraph -->
 
+
 <!-- wp:paragraph -->
-<p>o Merge the <strong>gutenberg-mobile</strong> PR to <code>trunk</code>. Use "Create a merge commit" option when merging, otherwise there could be conflicts between <code>trunk</code> and release branch in the next release. WARNING: Do not merge the <strong>gutenberg</strong> PR into <code>trunk</code> at this point.</p>
+<p>o <a href="https://github.com/wordpress-mobile/gutenberg-mobile/releases/new?tag=vX.XX.X&amp;target=release/X.XX.X&amp;title=Release%20X.XX.X">Create a new gutenberg-mobile GitHub Release</a>. Include a list of changes in the Release description.</p>
```

Comment:
@mkevins  mentions the order of tagging the release in the [issue](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/issues/90)

Instead we can create the release before merging. Just need to make sure the target branch of the release is not trunk.

Reply:
Minor issue: I think it would be good to move the "verify the <code>gutenberg</code> ref within the gutenberg-mobile release branch is pointed to the latest commit in the gutenberg release branch." instruction from the start of this "Integrate the Release" section (i.e., from before the gb release is tagged) to just before this step (i.e., to after the gb release is tagged). That way it could reference making sure that the submodule is pointing to the gb release tag, and it would reduce how often the steps are jumping back and forth between the two repos.

This is not a big deal, and it is outside the scope of what you're trying to focus on with this PR, so feel free to leave out this change.